.github/workflows/sonarqube.yml

```diff
@@ -0,0 +1,90 @@
+name: Release
+
+env:
+  SONAR_TOKEN: ${{ secrets.SONARQUBE_TOKEN }}
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
```

Comment:
same lines of which were removed from the release.yml workflow.

```suggestion
```

Reply:
@ipowellBT Could explain why is also needed in the sonarqube.yml file?

Reply:
From BIPS-17870

> Please note that [workflows triggered by ](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#about-workflow-runs-from-public-forks)pull_request_target events are run in the context of the base branch. Since the base branch is considered trusted, workflows triggered by these events will always run, regardless of approval settings.